### PR TITLE
New concept doc to explain how to build notificationChannelurl

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**

### CODE REVIEW

File: concepts/education-build-notificationchannelurl.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Build notificationChannelUrl"
+description: "Learn how to build the notificationChannelUrl property value for assignments"
+ms.localizationpriority: medium
+author: "cristobal-buenrostro"
+ms.prod: "education"
+doc_type: conceptualPageType
+---
+
+# Build notificationChannelUrl property value using the Microsoft Graph API
```
Comment (sharad-sharma-msft) [inline concepts/education-build-notificationchannelurl.md:10]: I would suggest adding the answer to "why". Why does one need this url? what are the use cases?

File: concepts/education-build-notificationchannelurl.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+This article describes how to use the education API in Microsoft Graph to build the `notificationChannelUrl` property for an [educationAssignment](/graph/api/resources/educationassignment).
+
+## Prerequisites
+
+Before you can build the property, you must have identified the team for the current **assignment** and the channel name.
```
Comment (sharad-sharma-msft) [inline concepts/education-build-notificationchannelurl.md:16]: will some screenshots here be more helpful?

File: concepts/education-build-notificationchannelurl.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+The following example shows a **notificationChannelUrl** based on this format.
+
+```http
+https://graph.microsoft.com/v1.0/teams/72a7baec-c3e9-4213-a850-f62de0adad5f/channels/19:jb2-ckDy2jONyW6ElO1phAVD5cTjuswYgoumI0oxrUw1@thread.tacv2
+```
+
+### Step 3 - Set the notificationChannelUrl property for the assignment
+
+You have now successfully built the url, you can set it in any of these calls
```
Comment (sharad-sharma-msft) [inline concepts/education-build-notificationchannelurl.md:73]: set -> consume

File: concepts/toc.yml [hunk 4]
```diff
@@ -256,6 +256,8 @@ items:
             href: education-upload-resource-overview.md
           - name: Assignment and submission state transitions
             href: assignments-submissions-states-transition.md            
+          - name: Build notificationChannelUrl value
```
Comment (sharad-sharma-msft) [inline concepts/toc.yml:259]: value ->property value

**Review by sharad-sharma-msft — CHANGES_REQUESTED** [review]

File: concepts/education-build-notificationchannelurl.md (hunk 3 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:73]: done

File: concepts/toc.yml (hunk 4 above)
Comment (cristobal-buenrostro) [reply concepts/toc.yml:259]: done

File: concepts/education-build-notificationchannelurl.md (hunk 1 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:10]: done

File: concepts/education-build-notificationchannelurl.md (hunk 2 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:16]: done

File: concepts/education-build-notificationchannelurl.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+description: "Learn how to build the notificationChannelUrl property value for assignments"
+ms.localizationpriority: medium
+author: "cristobal-buenrostro"
+ms.prod: "education"
+doc_type: conceptualPageType
+---
+
+# Build notificationChannelUrl property value using the Microsoft Graph API
+
+This article describes how to use the education API in Microsoft Graph to build the `notificationChannelUrl` property for an [educationAssignment](/graph/api/resources/educationassignment).
```
Comment (sharad-sharma-msft) [inline concepts/education-build-notificationchannelurl.md:12]: should you also mention here the data type for this property?

File: concepts/education-build-notificationchannelurl.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+The following example shows a **notificationChannelUrl** based on this format.
+
+```http
+https://graph.microsoft.com/v1.0/teams/72a7baec-c3e9-4213-a850-f62de0adad5f/channels/19:jb2-ckDy2jONyW6ElO1phAVD5cTjuswYgoumI0oxrUw1@thread.tacv2
+```
+
+### Step 4 - Consume the notificationChannelUrl property for the assignment
+
+You have now successfully built the url, you can consume it in any of these calls
+* [Update educationAssignmentDefaults](/graph/api/educationassignmentdefaults-update)
```
Comment (sharad-sharma-msft) [inline concepts/education-build-notificationchannelurl.md:127]: shall we also include a link to this article in these pages?

**Review by sharad-sharma-msft — CHANGES_REQUESTED** [review]

File: concepts/education-build-notificationchannelurl.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+This property is used to specify the default Teams channel to which notifications will be sent. Default value is null.
+
+## Prerequisites
+
+Before you can build the property, you must have identified the team for the current **assignment** and the channel name.
+
+**Team**
+
+![Identifying team](./images/notificationchannel-team.png)
```
Comment (isvargasmsft) [inline concepts/education-build-notificationchannelurl.md:22]: Please add proper alt text to the images. Please keep in mind users that rely on screen readers.

File: concepts/education-build-notificationchannelurl.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+**Channel**
+
+![Identifying channel](./images/notificationchannel-channel.png)
+
+## notificationChannelUrl property
```
Comment (isvargasmsft) [inline concepts/education-build-notificationchannelurl.md:28]: I'd suggest a more descriptive title for this header. Something that let's the readers know what is this section about

File: concepts/education-build-notificationchannelurl.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+The following steps describe how to build the property value.
+
+### Step 1 - Get teamd id
```
Comment (isvargasmsft) [inline concepts/education-build-notificationchannelurl.md:32]: teamd: is this a typo?

File: concepts/education-build-notificationchannelurl.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+Make a GET request with the team name.
```
Comment (isvargasmsft) [inline concepts/education-build-notificationchannelurl.md:33]: Please provide a better description for the step. Explain the goal of the step (get the team id) and how to achieve it (using a GET request with filter by name). Add a clarification to the user: if you already have the team id, you can skip this step.

Comment (isvargasmsft) [reply concepts/education-build-notificationchannelurl.md:33]: Apply this to the step 2 as well.

File: concepts/education-build-notificationchannelurl.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+| Placeholder | Description | Example |
+|:--|:--|:--|
+| `{team-id}` | Same team ID from the response in step 1. It is the team which current assignment belongs to. | 72a7baec-c3e9-4213-a850-f62de0adad5f |
+| `{channel-id}` | Item ID from the response body obtained in step 2. | 19:jb2-ckDy2jONyW6ElO1phAVD5cTjuswYgoumI0oxrUw1@thread.tacv2 |
+
+The following example shows a **notificationChannelUrl** based on this format.
+
+```http
+https://graph.microsoft.com/v1.0/teams/72a7baec-c3e9-4213-a850-f62de0adad5f/channels/19:jb2-ckDy2jONyW6ElO1phAVD5cTjuswYgoumI0oxrUw1@thread.tacv2
+```
```
Comment (isvargasmsft) [inline concepts/education-build-notificationchannelurl.md:122]: This step is not clear. How does the developer construct the property?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please provide better descriptions to the steps. Put yourself in the shoes of a developer who is starting with Graph.

File: concepts/education-build-notificationchannelurl.md (hunk 5 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 7 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 9 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 10 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:33]: done

File: concepts/education-build-notificationchannelurl.md (hunk 11 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: it is crystal clear =s

it is just to replace the placeholders with the values obtained in previous calls, the table gives details on that.
I just changed a bit the paragraph format to make it more understandable.

File: concepts/education-build-notificationchannelurl.md (hunk 8 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 6 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:131]: good idea, done

**Review by sharad-sharma-msft — APPROVED** [review]

**Review by isvargasmsft — APPROVED** [review]

File: concepts/education-build-notificationchannelurl.md [hunk 12]
```diff
@@ -0,0 +1,247 @@
+---
+title: "Build notificationChannelUrl for an education assignment"
+description: "How to build the notificationChannelUrl property value for assignments"
+ms.localizationpriority: medium
+author: "cristobal-buenrostro"
+ms.prod: "education"
+doc_type: conceptualPageType
+---
+
+# Build notificationChannelUrl for an education assignment using the Microsoft Graph API
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:10]: Can we update the title to more clearly reflect the action that the user can accomplish? For example, "Specify the default channel for education assignment notifications using the Microsoft Graph API"

File: concepts/education-build-notificationchannelurl.md [hunk 13]
```diff
@@ -0,0 +1,247 @@
...
+
+### Pre-requisite 2: Identify the channel
+
+![Once you have selected the team, select the appropiate channel from the Channels section](./images/notificationchannel-channel.png)
+
+## Build the notificationChannelUrl property value
+
+The following steps describe how to build the property value.
+
+### Step 1 - Get the team id based on your team name
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:32]: Please change id to ID when you're referring to an identifier and not specifically an **id** property.

File: concepts/education-build-notificationchannelurl.md [hunk 14]
```diff
@@ -0,0 +1,247 @@
...
+
+The following example shows a **notificationChannelUrl** based on this format.
+
+```http
+https://graph.microsoft.com/v1.0/teams/72a7baec-c3e9-4213-a850-f62de0adad5f/channels/19:jb2-ckDy2jONyW6ElO1phAVD5cTjuswYgoumI0oxrUw1@thread.tacv2
+```
+
+### Step 4 - Assign the value to the notificationChannelUrl property for the assignment
+
+You have now successfully built the url, it's time to assign the value to the property. You can perform this operation by updating either the **educationAssignment** or **educationAssignmentDefaults** resources.
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:130]: Please revise the first sentence to be two separate sentences and capitalize url: "You have now successfully built the URL. It's time to assign the value to the property."

File: concepts/toc.yml [hunk 15]
```diff
@@ -256,6 +256,8 @@ items:
             href: education-upload-resource-overview.md
           - name: Assignment and submission state transitions
             href: assignments-submissions-states-transition.md            
+          - name: Build notificationChannelUrl property value
```
Comment (Lauragra) [inline concepts/toc.yml:259]: Update the TOC entry to reflect the update to the topic title.

File: concepts/education-build-notificationchannelurl.md [hunk 16]
```diff
@@ -0,0 +1,247 @@
...
+
+This article describes how to use the education API in Microsoft Graph to build the `notificationChannelUrl` string property for an [educationAssignment](/graph/api/resources/educationassignment).
+
+This property is used to specify the default Teams channel to send notifications about the **assignment**. Default value is null.
+
+## Prerequisites
+
+Before building the property, identify the corresponding team for the **assignment** and the name of the channel.
+
+### Pre-requisite 1: Identify the team
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:20]: I don't know that we need subheadings here; they take a lot of vertical space in the article. Please change this to a sentence. Please also use text in addition to the illustration with alt text to explain the step. Suggested text to replace the H3 heading: "To identify the team for the assignment, in the left menu in Teams, click **Teams** and then select the appropriate team."

File: concepts/education-build-notificationchannelurl.md [hunk 17]
```diff
@@ -0,0 +1,247 @@
...
+
+# Build notificationChannelUrl for an education assignment using the Microsoft Graph API
+
+This article describes how to use the education API in Microsoft Graph to build the `notificationChannelUrl` string property for an [educationAssignment](/graph/api/resources/educationassignment).
+
+This property is used to specify the default Teams channel to send notifications about the **assignment**. Default value is null.
+
+## Prerequisites
+
+Before building the property, identify the corresponding team for the **assignment** and the name of the channel.
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:18]: I think you're just referring to the assignment here, not the resource type, so please remove the bold formatting.

File: concepts/education-build-notificationchannelurl.md [hunk 18]
```diff
@@ -0,0 +1,247 @@
...
+
+### Pre-requisite 1: Identify the team
+
+![In the left vertical menu in Microsoft Teams, click on Teams option and select the desired team](./images/notificationchannel-team.png)
+
+### Pre-requisite 2: Identify the channel
```
Comment (Lauragra) [inline concepts/education-build-notificationchannelurl.md:24]: Replace the heading with: "Identify the appropriate channel within the team that you selected."

File: concepts/education-build-notificationchannelurl.md (hunk 12 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 13 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 14 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md:130]: done

File: concepts/toc.yml (hunk 15 above)
Comment (cristobal-buenrostro) [reply concepts/toc.yml]: done

File: concepts/education-build-notificationchannelurl.md (hunk 16 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 17 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

File: concepts/education-build-notificationchannelurl.md (hunk 18 above)
Comment (cristobal-buenrostro) [reply concepts/education-build-notificationchannelurl.md]: done

**Pull request merged**